mmv1/products/compute/TargetHttpProxy.yaml

```diff
@@ -98,3 +107,12 @@ properties:
       This field only applies when the forwarding rule that references
       this target proxy has a loadBalancingScheme set to INTERNAL_SELF_MANAGED.
     default_from_api: true
+  - !ruby/object:Api::Type::Integer
+    name: 'httpKeepAliveTimeoutSec'
```

Comment:
This needs min_version: beta tag also, or else the field itself will show up in the GA provider

Reply:
Makes sense, added the tag to the new field.